activemq_xml/datadog_checks/activemq_xml/data/conf.yaml.example

```diff
@@ -155,7 +155,7 @@ instances:
     ## a domain that is used in the check. For NTLM Auth, append
     ## the username to domain, not as the `username` parameter.
     #
-    # ntlm_domain: <NTLM_DOMAIN>\<USERNAME>
+    # ntlm_domain: <NTLM_DOMAIN>/<USERNAME>
```

Comment:
Master is the previous version

https://github.com/DataDog/integrations-core/blob/87e6f39df8981fc73ea3fe741c60aed467aa2581/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/http.yaml#L86